testcases/cloud_admin/modify_instance_type.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+#
+# Description: Test the euca-modify-instance-type command works properly.
+#
+# NOTE: This testcase DOES NOT work with credentials file (credpath and eucarc) it only
+#       allows password/config_data for the time being.
+#
+from eucaops import Eucaops
+from eutester.eutestcase import EutesterTestCase
+
+import re
+import random
+
+class instance_type():  # mimicing a struct, I think this is the preferred way in python
```

Comment:
Class names should be CapWords per PEP8:
https://www.python.org/dev/peps/pep-0008/#class-names